src/NNlib.jl

```diff
@@ -1,14 +1,19 @@
 module NNlib
+using Pkg
```

Comment:
do we need this?

Reply:
Yes, for the error message to say what platform you are on. https://github.com/FluxML/NNlib.jl/pull/204/files#diff-04ed3dd551723d9193b5371386dea323R14